go.mod

```diff
@@ -50,23 +50,23 @@ require (
 	github.com/pion/datachannel v1.5.9
 	github.com/pion/ice/v2 v2.3.36
 	github.com/pion/logging v0.2.2
-	github.com/pion/sctp v1.8.33
+	github.com/pion/sctp v1.8.34
 	github.com/pion/stun v0.6.1
 	github.com/pion/webrtc/v3 v3.3.4
```

Comment:
Can you run `go mod tidy`

The dependency on v3 should be cleaned up 

<img width="302" alt="image" src="https://github.com/user-attachments/assets/674656d8-51f8-4046-97f7-94626ceb6950" />

Reply:
Done in both main package and `test-plan`.